examples/playback_capture.rs

```diff
@@ -15,7 +15,7 @@ fn main() {
     let in_b = client.register_port("rust_in_r", j::AudioInSpec::default()).unwrap();
     let mut out_a = client.register_port("rust_out_l", j::AudioOutSpec::default()).unwrap();
     let mut out_b = client.register_port("rust_out_r", j::AudioOutSpec::default()).unwrap();
-    let process_callback = move |ps: &j::ProcessScope| -> jack::JackControl {
+    let process_callback = move |_: &j::WeakClient, ps: &j::ProcessScope| -> jack::JackControl {
```

Comment:
I'm not sure what you mean by `WeakClient` implementing `JackHandler`, I thought it just gets passed as a callback parameter?